sdk/storage/storage-blob/src/BlobBatch.ts

```diff
@@ -49,14 +49,16 @@ export interface BatchSubRequest {
 }
 
 /**
- * A BatchRequest represents a based class for BatchDeleteRequest and BatchSetTierRequest.
+ * A BlobBatch represents an aggregated set of operations on blobs.
+ * Currently, only delete and setAccessTier are supported.
```

Comment:
Would better add comments like: only one kind of operation is supported per batch. This might be applicable to other langs.